yt/yt/server/controller_agent/config.cpp

```diff
@@ -706,6 +708,10 @@ void TDockerRegistryConfig::Register(TRegistrar registrar)
         .Default(true);
 
     registrar.Postprocessor([&] (TDockerRegistryConfig* options) {
+        if (!options->InternalRegistryAddress.empty()) {
```

Comment:
Checking string on emptiness is a bad practice, let make it optional

Reply:
Why?
FQDN cannot be empty.